docs/access.md

```diff
@@ -1,20 +1,118 @@
+
+*   **Admins:** Have complete access and control over all resources within the organization, including managing teams, instances, spaces, and organization settings.
+*   **Managers:** Can manage teams, instances, and spaces but may have limitations on modifying organization-level settings.
+*   **Members:** Can be granted access to specific resources (teams, instances, spaces) based on assignments. Default access might be limited.
+*   **Guests:** Intended for external collaborators with limited access, typically restricted to specific instances or spaces they are explicitly invited to.
+
+### Team roles
+
+*   **Admins:** Can add/remove team members, define member roles within the team context, and manage team resources or settings. Can typically perform any action a member can.
+*   **Members:** Can access resources granted to the team (e.g., specific instances or spaces).
+
+### Instance roles
+
+*   **Admins:** Can add/remove collaborators from the instance, define collaborator roles within the instance, and manage instance settings. For data access, automatically receive write access to the default "All" space only.
+*   **Read collaborators:** Automatically receive read access to the default "All" space only.
+*   **Write collaborators:** Automatically receive write access to the default "All" space only.
+
+**Note:** Permissions for spaces other than the default "All" space must be managed separately and independently of the instance collaborator role.
+
+### Space roles
+
+*   **Admins:** Have full control over the specific space, including managing permissions and content within that space.
+*   **Read collaborators:** Can read data and view resources within that specific space across accessible instances.
+*   **Write collaborators:** Can read, add, and modify data or resources within that specific space across accessible instances.
+
+## Practical implementation example
+
+A typical organization might structure its spaces within instances like this:
+
+1.  **`"All"` dpace (default):**
+    *   **Purpose:** Contains common assets like ontologies, tutorials, and non-sensitive datasets accessible to everyone within the instance.
+    *   **Access:** Generally broad read access for instance collaborators.
+
+2.  **`"Curation"` space:**
+    *   **Purpose:** Stores sensitive curated data requiring stricter access control.
+    *   **Access example:**
+        *   `"Curation Team"` has write access.
+        *   `"ML Team"` has read access.
+        *   No access granted to other teams by default.
+
+3.  **`"ML"` space:**
+    *   **Purpose:** Contains machine learning models, development resources, and potentially experimental data.
+    *   **Access example:**
+        *   Only `"ML Team"` has access (read/write as needed).
+        *   Completely isolated from other teams unless explicitly granted.
+
+## Getting started guide
+
+Follow these steps to configure your access control structure:
+
+### 1. Set up your organization structure
```

Comment:
I would focus on 3 common use cases and provide a brief description for how to achieve this using screenshots.

First use case: Add or remove an instance collaborator.

Second use case: Make space within an instance to restrict access.

Third use case: Make a team to bulk-manage access for multiple users.